cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -198,7 +198,7 @@ private void doInitialize(File programOptionFile) throws Exception {
     Injector injector = Guice.createInjector(createModule(cConf, hConf, programOptions, programRunId));
 
     // Setup the proxy selector for in active monitoring mode
-    if (cConf.getBoolean(io.cdap.cdap.common.conf.Constants.RuntimeMonitor.ACTIVE_MONITORING)) {
+    if (cConf.getBoolean(io.cdap.cdap.common.conf.Constants.RuntimeMonitor.ACTIVE_MONITORING, false)) {
```

Comment:
I think this should be true.
https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RemoteExecutionJobMain.java#L142